src/main.cpp

```diff
@@ -2927,7 +2920,7 @@ static void NotifyHeaderTip() {
  * or an activated best chain. pblock is either NULL or a pointer to a block
  * that is already loaded (to avoid loading it again from disk).
  */
```

Comment:
Agreed, will do. In case it's not clear, the intention is to allow us to avoid skipping the network altogether for utility functions. Beyond the startup stuff where it's currently skipped, in the future it may be interesting to do things like on-the-fly reindexing without having to shut down.